spec/index.bs

```diff
@@ -3278,7 +3278,7 @@ once all previously submitted operations using it are complete.
 
                 [=Content timeline=] steps:
 
-                1. If |this| is mapped, call |this|.{{GPUBuffer/unmap()}}.
```

Comment:
What do you think of just calling `unmap()` unconditionally? It already checks the state of the buffer (after we resolved #3251), and it does so more rigorously than saying "is mapped or pending".

Reply:
Sounds good to me. Updated, thanks.